instruqt-tracks/terraform-cloud-azure-v2/02-terraform-cloud-setup/check-workstation

```diff
@@ -1,9 +1,119 @@
 #!/bin/bash -l
+# Copyright (c) HashiCorp, Inc.
 set -euxvo pipefail
 
+# Get TFC token and organization
+# these jq commands may fail, that's okay, we account for 
+# that later, so temporarily disable set -e
+set +e
+# Check to make sure these are the right paths
+TFC_TOKEN=$(jq -r '.credentials."app.terraform.io".token' /root/.terraform.d/credentials.tfrc.json 2>/dev/null)
+set -e
+
 # Create /tmp/skip-check to disable this check
+# This /tmp/skip-check file is only necessary for the instruqt track test command 
+# When running this track as a customer/participant, skipping is disabled so it
+# this code will not have an affect.
 if [ -f /tmp/skip-check ]; then
     rm /tmp/skip-check
 fi
 
+# Save the Terraform Org and Workspace name as env variables 
+export ORG=$(grep organization /root/terraform-cloud/terraform.tfvars | cut -d '"' -f2)
+export WORKSPACE=$(grep workspace /root/terraform-cloud/terraform.tfvars | cut -d '"' -f2)
+
+# These are considered runtime variables, which allow you to use dynamic content in the challenge 
+# assignments and lifecycle scripts. 
+# In order to register a runtime variable in a host's lifecycle script, use the following command: 
+# agent variable set {KEY} {VALUE}
+agent variable set TF_ORG $ORG
+agent variable set TF_WORKSPACE $WORKSPACE
+
+cp /root/hashicat-azure/remote_backend.tf.example /root/hashicat-azure/remote_backend.tf
+
+cd /root/hashicat-azure
+sed -i "s/YOUR_ORGANIZATION/$ORG/g" remote_backend.tf
+sed -i "s/YOUR_WORKSPACE/$WORKSPACE/g" remote_backend.tf
+
+cp /root/hashicat-azure/terraform.tfvars.example /root/hashicat-azure/terraform.tfvars
+
+# This was originally going into the terraform-api directory... might need to check on this in the API challenge
+cd /root/terraform-cloud
+sed -i "s/YOUR_ORGANIZATION/$ORG/g" terraform.tfvars # This was originally looking in terraform.tfvars.example
+sed -i "s/YOUR_WORKSPACE/$WORKSPACE/g" terraform.tfvars
+# mv terraform.tfvars.example terraform.tfvars
+# The above command will get rid of the terraform.tfvars.example file, but when
+# we run the check script more than once, it cannot find that file bc it no longer
+# exists. However, to keep the consistency between code blocks, we kept this the 
+# same as above where we search and replace in the remote_backend.tf file. 
+
+# Store the ORG in /root/.bashrc
```

Comment:
This is the only blocking comment I have.

Reply:
Follow up questions:

Should I also remove these lines in the AWS and GCP tracks?
Why do the AWS and GCP tracks not have a cleanup script in challenge 02 but the Azure track does?

Reply:
- In this PR, you can leave AWS and GCP tracks as they are. Just remove what was added to Azure here.
- They never got removed from AWS and GCP because the initial ticket, VCDL-125, was identifying a problem with the Azure track, and it was only after it was fixed there I realized we need to do the same thing in AWS and GCP. We just haven't gotten around to it. 

If you want to grab VCDL-125 after this is done, and finish what I did for Azure in that ticket to AWS and GCP, that would be fine --- but we can handle that in that ticket, and leave this PR just with what is necessary to complete VCDL-103.